src/visibility.js

```diff
@@ -203,8 +188,6 @@ class Visibility extends BasePlugin {
     this._appTargetContainer = null;
     this._floatingContainer = null;
     this._floatingPoster = null;
```

Comment:
I think you need to destroy the viewabilityManager

Reply:
the viewability manager is a player service, it is in charge of destroying it.

Reply:
remove the specific handler from it? if the plugin is not necessary anymore it'll keep the listener

Reply:
When Kaltura is destroyed we call intersection observer disconnect which remove all listeners.
Is there a scenario where we destroy only a plugin? If so then we need to to add an unobserve to the plugin destroy.

Reply:
@RoyBregman we should have an unobserve API as we discussed

Reply:
We have an unObserve, i just didn't call it cause Kaltura player destroys the plugin when it is destroyed.
Anyway I added the unObserve call also here.